includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -514,6 +532,15 @@ function( $scope ) {
 		);
 		$this->set_granted_scopes( $scopes );
 
+		// If using the proxy, these values can reliably be set at this point because the proxy already took care of
+		// them.
+		// TODO: In the future, once the old authentication mechanism no longer exists, this should be resolved in
+		// another way.
+		if ( $this->using_proxy() ) {
+			$this->user_options->set( Verification::OPTION, 'verified' );
+			$this->options->set( Search_Console::PROPERTY_OPTION, trailingslashit( $this->context->get_reference_site_url() ) );
```

Comment:
Escape raw for DB
```suggestion
			$this->options->set( Search_Console::PROPERTY_OPTION, esc_url_raw( trailingslashit( $this->context->get_reference_site_url() ) ) );
```

Reply:
Why that? We haven't done that so far on this option, and I'm not sure this is recommended.

Reply:
Makes sense, but we haven't done it so far, and we also don't do it in the other code where that option can be set. I'd prefer to not change it at this point so briefly before release, as there might be implications related to how the URL is escaped before storage.